scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -192,6 +237,15 @@ namespace <%= namespace %> {
             );
         }
 
+        private void AvailableShippingRatesPoll(Action<List<string>, string> callback) {
+            PollTillReady(
+                checkout => {
+                    return checkout.availableShippingRates().ready();
+                },
+                callback
```

Comment:
`PollTillReady` will handle updating the checkout. Here we are passing the `callback` to `PollTillReady`